### ISSUE TITLE
Rare client error

### ISSUE THREAD

**Issue by JulianKropp** [bigbluebutton-bot/bbb-translation-bot#13]
client:
```go
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0xa0 pc=0xb91b45]

goroutine 89 [running]:
github.com/bigbluebutton-bot/bigbluebutton-bot/pad.(*ChangesetClient).StopChangesetServer(...)
	/home/user/go/pkg/mod/github.com/bigbluebutton-bot/bigbluebutton-bot@v0.1.1/pad/changeset.go:205
github.com/bigbluebutton-bot/bigbluebutton-bot/pad.(*Pad).onDisconnect(0xc000422000, 0x0?)
	/home/user/go/pkg/mod/github.com/bigbluebutton-bot/bigbluebutton-bot@v0.1.1/pad/padManager.go:215 +0x65
reflect.Value.call({0xc20840?, 0xc00061a880?, 0x4df113?}, {0xd83aca, 0x4}, {0xc0002dde88, 0x1, 0xc0003961c0?})
	/usr/local/go/src/reflect/value.go:596 +0xca6
reflect.Value.Call({0xc20840?, 0xc00061a880?, 0x40bc40?}, {0xc0002dde88?, 0x4?, 0xc0002ddeb8?})
	/usr/local/go/src/reflect/value.go:380 +0xb9
github.com/bigbluebutton-bot/golang-socketio.(*caller).callFunc(0xc000619500, 0xd8c565?, {0xc09020?, 0x14e63a0?})
	/home/user/go/pkg/mod/github.com/bigbluebutton-bot/golang-socketio@v0.0.0-20231114184021-0b9dca893d46/caller.go:74 +0x18c
github.com/bigbluebutton-bot/golang-socketio.(*methods).callLoopEvent(0xc00019c540, 0xc00019c570, {0xd8c565, 0xd})
	/home/user/go/pkg/mod/github.com/bigbluebutton-bot/golang-socketio@v0.0.0-20231114184021-0b9dca893d46/handler.go:85 +0xd4
github.com/bigbluebutton-bot/golang-socketio.closeChannel(0xc00019c570, 0xc00019c540, {0xd897f7?, 0xa?, 0xc00019c588?})
	/home/user/go/pkg/mod/github.com/bigbluebutton-bot/golang-socketio@v0.0.0-20231114184021-0b9dca893d46/loop.go:111 +0x125
github.com/bigbluebutton-bot/golang-socketio.inLoop(0xc00019c570, 0xc00019c540)
	/home/user/go/pkg/mod/github.com/bigbluebutton-bot/golang-socketio@v0.0.0-20231114184021-0b9dca893d46/loop.go:125 +0x22d
created by github.com/bigbluebutton-bot/golang-socketio.(*Client).Dial in goroutine 1
	/home/user/go/pkg/mod/github.com/bigbluebutton-bot/golang-socketio@v0.0.0-20231114184021-0b9dca893d46/client.go:61 +0xa5
exit status 2
```

**Comment by JulianKropp** [comment]
```
changeset-server      | {
changeset-server      |   wrappers_: null,
changeset-server      |   messageId_: undefined,
changeset-server      |   arrayIndexOffset_: -1,
changeset-server      |   array: [
changeset-server      |     'notes in in',
changeset-server      |     'Blut sind gar nicht. Gar nicht, gar nicht, gar nicht.',
changeset-server      |     '|1+1'
changeset-server      |   ],
changeset-server      |   pivot_: 1.7976931348623157e+308,
changeset-server      |   convertedPrimitiveFields_: {}
changeset-server      | }
bot                   | Generated changeset:  Z:c>16-b+1h$Blut sind gar nicht. Gar nicht, gar nicht, gar nicht.
bot                   | {"type":"COLLABROOM","component":"pad","data":{"type":"USER_CHANGES","baseRev":1432,"changeset":"Z:c\u003e16-b+1h$Blut sind gar nicht. Gar nicht, gar nicht, gar nicht.","apool":{"numToAttrib":{"0":["author","a.zFI8YhYVDYQ537GW"]},"nextNum":1}}}
bot                   | map[data:map[newRev:1433 type:ACCEPT_COMMIT] type:COLLABROOM]
bot                   | ACCEPT_COMMIT
bot                   | plaintext bytes:  [80 73 78 71]
bot                   | encryptedToken:  [82 178 116 75]
bot                   | map[data:map[payload:map[action:cursorPosition authorId:a.zFI8YhYVDYQ537GW authorName:Bot locationX:76861 locationY:0 padId:g.kjCzJZzdFNkXPpTF$en] type:CUSTOM] type:COLLABROOM]
bot                   | CUSTOM
bot                   | sent ping message!
bot                   | received pong response!
bot                   | plaintext bytes:  [80 73 78 71]
bot                   | encryptedToken:  [82 178 116 75]
bot                   | MSG gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar 
bot                   | MSG <8����q�j7ťDgar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar
bot                   | TCP message event: <8����q�j7ťDgar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar
bot                   | Old text:  Blut sind gar nicht. Gar nicht, gar nicht, gar nicht.
bot                   | New text:  <8qj7ťDgar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar
bot                   | TCP message event: gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar 
bot                   | Old text:  Blut sind gar nicht. Gar nicht, gar nicht, gar nicht.
bot                   | New text:  gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar 
changeset-server      | {
changeset-server      |   wrappers_: null,
changeset-server      |   messageId_: undefined,
changeset-server      |   arrayIndexOffset_: -1,
changeset-server      |   array: [
changeset-server      |     'Blut sind gar nicht. Gar nicht, gar nicht, gar nicht.',
changeset-server      |     '<8\x1B=q\x07j7ťDgar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar',
changeset-server      |     '|1+1'
changeset-server      |   ],
changeset-server      |   pivot_: 1.7976931348623157e+308,
changeset-server      |   convertedPrimitiveFields_: {}
changeset-server      | }
changeset-server      | {
changeset-server      |   wrappers_: null,
changeset-server      |   messageId_: undefined,
changeset-server      |   arrayIndexOffset_: -1,
changeset-server      |   array: [
changeset-server      |     'Blut sind gar nicht. Gar nicht, gar nicht, gar nicht.',
changeset-server      |     'gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar ',
changeset-server      |     '|1+1'
changeset-server      |   ],
changeset-server      |   pivot_: 1.7976931348623157e+308,
changeset-server      |   convertedPrimitiveFields_: {}
changeset-server      | }
bot                   | Generated changeset:  Z:1i>qz-1h+sg$gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar 
bot                   | {"type":"COLLABROOM","component":"pad","data":{"type":"USER_CHANGES","baseRev":1433,"changeset":"Z:1i\u003eqz-1h+sg$gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar ","apool":{"numToAttrib":{"0":["author","a.zFI8YhYVDYQ537GW"]},"nextNum":1}}}
bot                   | Generated changeset:  Z:1i>u-1h+2b$<8qj7ťDgar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar
bot                   | {"type":"COLLABROOM","component":"pad","data":{"type":"USER_CHANGES","baseRev":1434,"changeset":"Z:1i\u003eu-1h+2b$\u003c8\u001b=q\u0007j7ťDgar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar nicht gar","apool":{"numToAttrib":{"0":["author","a.zFI8YhYVDYQ537GW"]},"nextNum":1}}}
bot                   | map[data:map[newRev:1434 type:ACCEPT_COMMIT] type:COLLABROOM]
bot                   | ACCEPT_COMMIT
bot                   | map[disconnect:badChangeset]
bot                   | Disconnected
bot                   | panic: runtime error: invalid memory address or nil pointer dereference
bot                   | [signal SIGSEGV: segmentation violation code=0x1 addr=0xa0 pc=0xb85445]
bot                   | 
bot                   | goroutine 7442 [running]:
bot                   | github.com/bigbluebutton-bot/bigbluebutton-bot/pad.(*ChangesetClient).StopChangesetServer(...)
bot                   |         /go/pkg/mod/github.com/bigbluebutton-bot/bigbluebutton-bot@v0.1.1/pad/changeset.go:205
bot                   | github.com/bigbluebutton-bot/bigbluebutton-bot/pad.(*Pad).onDisconnect(0xc0005af340, 0x0?)
bot                   |         /go/pkg/mod/github.com/bigbluebutton-bot/bigbluebutton-bot@v0.1.1/pad/padManager.go:215 +0x65
bot                   | reflect.Value.call({0xc12500?, 0xc0003fe390?, 0x1?}, {0xd72f1c, 0x4}, {0xc0004735f0, 0x1, 0xc000103a40?})
bot                   |         /usr/local/go/src/reflect/value.go:596 +0xca6
bot                   | reflect.Value.Call({0xc12500?, 0xc0003fe390?, 0x409d80?}, {0xc0004735f0?, 0x4?, 0xc000473620?})
bot                   |         /usr/local/go/src/reflect/value.go:380 +0xb9
bot                   | github.com/bigbluebutton-bot/golang-socketio.(*caller).callFunc(0xc00024d350, 0xd7b90a?, {0xbfb660?, 0x14c9020?})
bot                   |         /go/pkg/mod/github.com/bigbluebutton-bot/golang-socketio@v0.0.0-20231114184021-0b9dca893d46/caller.go:74 +0x18c
bot                   | github.com/bigbluebutton-bot/golang-socketio.(*methods).callLoopEvent(0xc0005af420, 0xc0005af450, {0xd7b90a, 0xd})
bot                   |         /go/pkg/mod/github.com/bigbluebutton-bot/golang-socketio@v0.0.0-20231114184021-0b9dca893d46/handler.go:85 +0xd4
bot                   | github.com/bigbluebutton-bot/golang-socketio.closeChannel(0xc0005af450, 0xc0005af420, {0xd790bc?, 0xa?, 0x1?})
bot                   |         /go/pkg/mod/github.com/bigbluebutton-bot/golang-socketio@v0.0.0-20231114184021-0b9dca893d46/loop.go:111 +0x125
bot                   | github.com/bigbluebutton-bot/golang-socketio.(*Client).Close(...)
bot                   |         /go/pkg/mod/github.com/bigbluebutton-bot/golang-socketio@v0.0.0-20231114184021-0b9dca893d46/client.go:73
bot                   | github.com/bigbluebutton-bot/bigbluebutton-bot/pad.(*Pad).Disconnect(...)
bot                   |         /go/pkg/mod/github.com/bigbluebutton-bot/bigbluebutton-bot@v0.1.1/pad/padManager.go:175
bot                   | github.com/bigbluebutton-bot/bigbluebutton-bot/pad.(*Pad).onMessage(0xc0005af340, 0x0?, {0xc534c0, 0xc0002f6360})
bot                   |         /go/pkg/mod/github.com/bigbluebutton-bot/bigbluebutton-bot@v0.1.1/pad/padManager.go:251 +0x117
bot                   | reflect.Value.call({0xc25960?, 0xc000468980?, 0xc0003fec60?}, {0xd72f1c, 0x4}, {0xc0001fbf08, 0x2, 0x7f8684844608?})
bot                   |         /usr/local/go/src/reflect/value.go:596 +0xca6
bot                   | reflect.Value.Call({0xc25960?, 0xc000468980?, 0x1468f20?}, {0xc0001fbf08?, 0x16?, 0x0?})
bot                   |         /usr/local/go/src/reflect/value.go:380 +0xb9
bot                   | github.com/bigbluebutton-bot/golang-socketio.(*caller).callFunc(0xc0004cf0b0, 0x1d?, {0xbea660?, 0xc0003fec70?})
bot                   |         /go/pkg/mod/github.com/bigbluebutton-bot/golang-socketio@v0.0.0-20231114184021-0b9dca893d46/caller.go:74 +0x18c
bot                   | github.com/bigbluebutton-bot/golang-socketio.(*methods).processIncomingMessage(0x14c9020?, 0xc0005af450, 0xc00028a7c0)
bot                   |         /go/pkg/mod/github.com/bigbluebutton-bot/golang-socketio@v0.0.0-20231114184021-0b9dca893d46/handler.go:114 +0x285
bot                   | created by github.com/bigbluebutton-bot/golang-socketio.inLoop in goroutine 88
bot                   |         /go/pkg/mod/github.com/bigbluebutton-bot/golang-socketio@v0.0.0-20231114184021-0b9dca893d46/loop.go:143 +0xd2
bot exited with code 2
```

**Comment by JulianKropp** [comment]
Fixed with [bbb-bot 0.1.4](https://github.com/bigbluebutton-bot/bigbluebutton-bot/releases/tag/v0.1.4)
